ubuntu_image/helpers.py

```diff
@@ -178,3 +214,19 @@ def get_default_sector_size():
         os.truncate(fp.name, 0)
         os.truncate(fp.name, MiB(1))
         return Device(fp.name).sectorSize
+
+
+def check_root_priviledge():
```

Comment:
For this kind of functions I would propose we exit more cleanly. I would prefer if the __main__.py parts were handling all the exit() functionality so I'd propose this method to throw an exception instead of doing exit(1). The exception handler in the main function could then print out any info and exit instead.